SUMMARY.md

```diff
@@ -11,21 +11,15 @@
 
 ## Wallets
 
-* [Arweave Web Extension Wallet](wallets/arweave-web-extension-wallet.md)
+* [Arweave.app Web Wallet](wallets/arweave-web-extension-wallet.md)
```

Comment:
suggestion: rename this file, otherwise the URL will be for the Arweave web extension